### PR TITLE
fix(ios): select a valid ios sim xcodebuild destination

### PR DESCRIPTION
**Pull request by cb1kenobi**
The desired iOS simulator's runtime may be associated with a different Xcode than the one building with, so simply pick any simulator that is compatible with the Xcode building the app since it won't matter when ioslib laucnhes the selected iOS Simulator.

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-27338

### CODE REVIEW

File: iphone/cli/commands/_build.js [hunk 1]
```diff
@@ -6776,9 +6776,27 @@ iOSBuilder.prototype.invokeXcodeBuild = function invokeXcodeBuild(next) {
 	];
 
 	if (this.simHandle) {
+		let dest;
+		const xcodeId = this.xcodeEnv.version + ':' + this.xcodeEnv.build;
+
+		// xcodebuild requires a -destination when building for iOS Simulator and it needs a
+		// simulator that is compatible with the selected Xcode version, so just pick one
+		for (const sim of Object.values(this.iosInfo.simulators.ios)) {
```
Comment (ewanharris) [inline iphone/cli/commands/_build.js]: `this.iosInfo.simulators.ios` is an object with the following structure
```
{
	'13.0': [ // array of sims ],
	'12.4': [ // array of sims ]
}
```

I think this wants to be something like the below
```js
for (const [ runtime, sims ] of Object.entries(this.iosInfo.simulators.ios)) {
	for (const sim of sims) {
		if (sim.supportsXcode[xcodeId]) {
			dest = `platform=iOS Simulator,id=${sim.udid},OS=${appc.version.format(sim.version, 2, 2)}`;
			break;
		}
	}
}
```

**Review by ewanharris — CHANGES_REQUESTED** [review]
I'm seeing a similar crash as Jenkins is locally, changing the code as commented appears to fix the crash

**Review by ewanharris — APPROVED** [review]
Looks good to me!

**Comment by ssekhri** [comment]
@cb1kenobi when trying to run a classic project on iOS 13 simulator, using XCode 11, I get the following error in both studio and cli
```
[INFO] :   Invoking xcodebuild
[DEBUG] :  Invoking: DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild build -target testClassic -configuration Debug -scheme testClassic -derivedDataPath /Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/DerivedData -UseNewBuildSystem=YES OBJROOT=/Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/build/Intermediates SHARED_PRECOMPS_DIR=/Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/build/Intermediates/PrecompiledHeaders SYMROOT=/Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/build/Products -destination "platform=iOS Simulator,id=441204EF-B8B5-4829-ADDA-F9141902FB23,OS=10.3" ONLY_ACTIVE_ARCH=1 APPC_PROJECT_DIR=/Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic
[TRACE] :  User defaults from command line:
[TRACE] :      IDEDerivedDataPathOverride = /Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/DerivedData
[TRACE] :      UseNewBuildSystem = YES
[TRACE] :  Build settings from command line:
[TRACE] :      APPC_PROJECT_DIR = /Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic
[TRACE] :      OBJROOT = /Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/build/Intermediates
[TRACE] :      ONLY_ACTIVE_ARCH = 1
[TRACE] :      SHARED_PRECOMPS_DIR = /Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/build/Intermediates/PrecompiledHeaders
[TRACE] :      SYMROOT = /Users/ssekhri/Documents/Appcelerator_Studio_Workspace/testClassic/build/iphone/build/Products
[ERROR] :  xcodebuild: error: Unable to find a destination matching the provided destination specifier:
[ERROR] :  		{ platform:iOS Simulator, id:441204EF-B8B5-4829-ADDA-F9141902FB23, OS:10.3 }
[ERROR] :  	The requested device could not be found because no available devices matched the request.
[ERROR] :  	Available destinations for the "testClassic" scheme:
[ERROR] :  		{ platform:iOS Simulator, id:476F296D-E2F6-40A0-B0C8-5D476FA70520, OS:10.3.1, name:iPad (5th generation) }
[ERROR] :  		{ platform:iOS Simulator, id:D2E9BE3F-9952-4FFF-BBB2-C316568A5CCB, OS:11.0.1, name:iPad (5th generation) }
[ERROR] :  		{ platform:iOS Simulator, id:DEA27542-9EE9-4226-B3DE-EAC5D0A38A81, OS:11.2, name:iPad (5th generation) }
[ERROR] :  		{ platform:iOS Simulator, id:B4E094E6-B70F-4A44-90C2-D1FB7C8EA8AF, OS:11.3, name:iPad (5th generation) }
[ERROR] :  		{ platform:iOS Simulator, id:5957845E-C101-4DFA-B8CA-56B987D3E555, OS:12.2, name:iPad (5th generation) }
[ERROR] :  		{ platform:iOS Simulator, id:B8D476C9-AEB0-4440-BDFB-CD96402438FA, OS:12.2, name:iPad (6th generation) }
[ERROR] :  		{ platform:iOS Simulator, id:A06532C4-3CB1-42DA-ADB6-EA9F0347799C, OS:9.3, name:iPad 2 }
[ERROR] :  		{ platform:iOS Simulator, id:E2644C01-3BA8-4D3C-9A1A-E3C8264433F7, OS:9.3, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:9AE43734-6717-4762-9A1F-0993EFBFC80C, OS:10.0, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:7DC09A42-8762-4BC0-B001-F422B21EDC9C, OS:10.3.1, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:CD6B65A5-6805-4C09-AD59-B0133E85B2B0, OS:11.0.1, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:84EAF295-F891-4968-8C3D-C3BC8769099C, OS:11.2, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:F29990FF-624D-4C80-8F4B-D52D0896878D, OS:11.3, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:351D7E45-9F0D-4C24-BF79-8C936481AF3A, OS:12.2, name:iPad Air }
[ERROR] :  		{ platform:iOS Simulator, id:089CDD53-07E6-4BF6-9952-DB1FD8C55D67, OS:12.2, name:iPad Air (3rd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:CD862665-076A-40B1-9079-1EA81D96A4F0, OS:13.0, name:iPad Air (3rd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:1C936B35-9355-4FB1-A628-6C7EE1F3E7EC, OS:9.3, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:9F7DB60B-F2D8-432E-B2E6-D3E19C04C468, OS:10.0, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:866120D8-9568-4F53-9BF4-9DD35A7B5755, OS:10.3.1, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:623D41A8-0CAA-4BF5-9302-180CE5B46F29, OS:11.0.1, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:1DF6B3A0-5833-43C2-8C02-B59650AEB46A, OS:11.2, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:5E5749DD-9732-4E04-BD44-3EDF63BBBAC3, OS:11.3, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:3EF1989B-ECE8-4454-8BF2-B7C1C7D3DE38, OS:12.2, name:iPad Air 2 }
[ERROR] :  		{ platform:iOS Simulator, id:328543A5-3255-46AB-95C7-7DC82D65ABB3, OS:9.3, name:iPad Pro }
[ERROR] :  		{ platform:iOS Simulator, id:0B4980A8-1C81-41C0-9721-AF39F4D7B800, OS:10.0, name:iPad Pro (9.7 inch) }
[ERROR] :  		{ platform:iOS Simulator, id:801BE701-F1A5-4C66-9DFD-94319CCDA548, OS:10.3.1, name:iPad Pro (9.7 inch) }
[ERROR] :  		{ platform:iOS Simulator, id:973392CA-B7D2-48E7-9CD7-BB0A3C3350AA, OS:11.0.1, name:iPad Pro (9.7-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:077B6D23-14EC-4E83-BA9E-2F223FDDE105, OS:11.2, name:iPad Pro (9.7-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:0A253233-B4E6-4398-B61F-06BB7087E7BF, OS:11.3, name:iPad Pro (9.7-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:F1906364-7DAE-4740-AB2E-7FE1414C66AD, OS:12.2, name:iPad Pro (9.7-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:1AAE8920-E6C3-4F3B-B1C8-5A7C87A7E273, OS:13.0, name:iPad Pro (9.7-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:88831FF6-BBE7-4AAC-9071-B10015E2062C, OS:10.3.1, name:iPad Pro (10.5-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:83E9AE15-18BE-49FB-833E-E0EEC2FF527D, OS:11.0.1, name:iPad Pro (10.5-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:977BD9AA-D4E1-40CA-B587-EE2259B29991, OS:11.2, name:iPad Pro (10.5-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:1B1C29A1-0D25-4139-A2B4-D37FEED6E922, OS:11.3, name:iPad Pro (10.5-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:E91D8B22-367B-4AE3-868E-A800215A387B, OS:12.2, name:iPad Pro (10.5-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:9711C31E-4119-46CD-8FDD-B61E2A40DADE, OS:12.2, name:iPad Pro (11-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:C06AB6B4-2CCF-456C-8167-A68EE2AF79BB, OS:13.0, name:iPad Pro (11-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:830413C9-A11E-4ED3-B4FA-192F04300D0B, OS:10.0, name:iPad Pro (12.9 inch) }
[ERROR] :  		{ platform:iOS Simulator, id:A4059A09-21F5-4EB8-AB58-762C8FB7E5BD, OS:10.3.1, name:iPad Pro (12.9 inch) }
[ERROR] :  		{ platform:iOS Simulator, id:48C69495-FBC1-4CD8-B4E6-C5BAC1DB0ECF, OS:11.0.1, name:iPad Pro (12.9-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:3EA44D3D-6CCB-452E-B397-90077A8852A9, OS:11.2, name:iPad Pro (12.9-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:5AA4B631-5BDC-4F0C-BE2E-8DEB92785490, OS:11.3, name:iPad Pro (12.9-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:78E99A35-2795-4A63-9752-CA4DABD0A0DC, OS:12.2, name:iPad Pro (12.9-inch) }
[ERROR] :  		{ platform:iOS Simulator, id:A6184F11-1A7A-4C1A-9D52-C0C4FE931AD1, OS:10.3.1, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:079B5DA0-3057-481D-8633-8EF896C589AB, OS:11.0.1, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:459650D9-BA88-4F0D-99EA-A10170D8BF60, OS:11.2, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:16EE9324-FDBF-45F1-B4E1-FDEB00BCF83C, OS:11.3, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:881BDF37-1F1E-4332-85C3-24B39E8B2A96, OS:12.2, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:328A554B-6671-440B-B11C-2D5B4B57D6F4, OS:12.2, name:iPad Pro (12.9-inch) (3rd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:256A2389-A254-41D8-8786-9ABAC3965275, OS:13.0, name:iPad Pro (12.9-inch) (3rd generation) }
[ERROR] :  		{ platform:iOS Simulator, id:D9A8252A-6A5B-49F9-A88F-CE22178D2DC5, OS:9.3, name:iPad Retina }
[ERROR] :  		{ platform:iOS Simulator, id:256261BC-22BF-4DCD-9F98-F6052469B964, OS:9.3, name:iPhone 4s }
[ERROR] :  		{ platform:iOS Simulator, id:837D821F-122F-437C-A1AC-5A2D496318C6, OS:9.3, name:iPhone 5 }
[ERROR] :  		{ platform:iOS Simulator, id:C624A90D-EDEF-49EA-A42F-EDAD30E85900, OS:10.0, name:iPhone 5 }
[ERROR] :  		{ platform:iOS Simulator, id:FF91071B-BEAF-4656-8CBE-3658D5C6ACFE, OS:10.3.1, name:iPhone 5 }
[ERROR] :  		{ platform:iOS Simulator, id:DCD41208-457F-40FC-946C-B2D21815E238, OS:9.3, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:3CEAA0F3-D838-4AB4-AE7B-D87F7BC3E074, OS:10.0, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:EF6ABB0B-C40F-4E0C-844A-604276A4F9AF, OS:10.3.1, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:4D3A3EA4-2D63-48CD-9A59-C1C5F2D7879A, OS:11.0.1, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:55D7B1A5-A9B2-4295-9A31-5292E4FA60D9, OS:11.2, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:880EC5BA-6629-47F6-A337-624CFE1D1589, OS:11.3, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:D0176736-C533-431F-BB22-CDAE0F41DB1D, OS:12.2, name:iPhone 5s }
[ERROR] :  		{ platform:iOS Simulator, id:878448EA-55B8-4862-9725-A92D536CA5BB, OS:9.3, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:68716B82-E27B-4FD1-B1C2-E3A0EDC95EF2, OS:10.0, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:EAD00A82-54A4-4FEB-A4D3-57D6B4E75137, OS:10.2, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:7BD3C2AF-421A-433C-BB9F-B443F8535FE1, OS:10.3.1, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:5D89B4AC-B26D-46D4-8482-3C77FB8B1782, OS:11.0.1, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:12CABAD4-0CD7-476F-85EB-3E23049C634E, OS:11.1, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:8021AE1A-80F6-4819-BB4E-6582DDC04605, OS:11.2, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:026C1C94-2E3D-43B0-9B5B-57AFE034899C, OS:11.3, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:8F3D949C-0307-4142-B2BA-6AF47B7B54F1, OS:12.2, name:iPhone 6 }
[ERROR] :  		{ platform:iOS Simulator, id:6D7DAA36-6FCB-4A8C-BCF1-540DB4199E07, OS:9.3, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:BB8F77DB-62D2-4A15-BD79-07D0753C83A4, OS:10.0, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:1D6C9D16-CA30-4A4F-9BAB-B5732BAE781B, OS:10.2, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:16AA41F5-1885-4C25-8BEB-AA540A0A443A, OS:10.3.1, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:C7BCE0E1-AA17-44A4-98DE-61660CC3FF35, OS:11.0.1, name:iPhone 6 Plus }
[ERROR] :  		{ platfor
[ERROR] :  m:iOS Simulator, id:BA3FAE7C-3298-4A51-A12D-231CEC3F7EB8, OS:11.1, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:7EF66D99-61EC-45E0-84BD-C99D13483300, OS:11.2, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:2BADC856-0D84-42CB-874B-01F0E7D2A84A, OS:11.3, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:F8F59373-0E51-434E-A487-8B38E0B1DC8B, OS:12.2, name:iPhone 6 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:E6CFDB9D-EFC9-4286-8B1D-071D08DAFE64, OS:9.3, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:6D02363A-4BA9-4AD6-80EF-5BE3A8902F69, OS:10.0, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:99A7C72B-0F6D-4DD7-BC7E-9FF6C32FCC9E, OS:10.2, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:77BC3564-CD0B-4FC6-B6B5-D8A6E89DCD80, OS:10.3.1, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:CDF50665-EEC0-4A76-931E-2B2DD86B290C, OS:11.0.1, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:8ACFAFA4-5E05-43FC-AFFB-819D192BB939, OS:11.1, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:85F6D00C-6D65-4738-8EAE-96703979BD34, OS:11.2, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:A3AABBFC-E64A-45B9-B775-0142D35D7CE8, OS:11.3, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:6400575F-339B-439E-AF40-341200EA953B, OS:12.2, name:iPhone 6s }
[ERROR] :  		{ platform:iOS Simulator, id:516AA219-7987-4759-9CAB-7DC3936A2222, OS:9.3, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:AB430870-7956-4CBF-B83C-C286F6E13656, OS:10.0, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:A69A6E9B-3C4A-4CC6-AB14-64E4F0BC1372, OS:10.2, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:2050EB3D-EA2B-4B07-A439-D5A0F3C37C6E, OS:10.3.1, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:CB3735B1-884A-4BFB-9F5E-B205CEB65345, OS:11.0.1, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:83648E79-810F-4E94-858D-285D322007E2, OS:11.1, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:C4E7FD4C-116D-42F7-821B-25C2C8CE06A7, OS:11.2, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:0155348B-9EFD-4AE7-95DA-55E7B800B055, OS:11.3, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:D929AD99-9611-444C-8951-05326BDB7DCB, OS:12.2, name:iPhone 6s Plus }
[ERROR] :  		{ platform:iOS Simulator, id:84A2C050-EDC5-47CE-BE72-694239DF116B, OS:10.0, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:4B6C351A-1712-4F8E-93C1-CB1EBEA558E8, OS:10.2, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:65A7C7B3-75BD-4DA7-9CBE-22579816D565, OS:10.3.1, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:53E0F799-482E-4D2A-8E9A-497C2BB3010F, OS:11.0.1, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:56999167-78A6-4FB8-96F8-275FAD0F425B, OS:11.1, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:481E8A1E-A135-4101-B0F8-642B2D18C2AE, OS:11.2, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:6A5E8609-114A-4681-9C2D-1B68CA3EBDE7, OS:11.3, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:64D0B40C-1596-4415-9C6C-FF37F1E62B83, OS:12.2, name:iPhone 7 }
[ERROR] :  		{ platform:iOS Simulator, id:70524A50-CF58-48A8-B9BF-2E0D74B89F4B, OS:10.0, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:8567E73F-9FFE-448D-9CCE-AA6B7957268F, OS:10.2, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:A1D69587-1875-44FC-8C8E-026BA23EDBE1, OS:10.3.1, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:35891D11-E982-497E-BC0C-8BEB7E78E00E, OS:11.0.1, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:177C670C-E980-458E-9215-FE50BE14F617, OS:11.1, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:344BE97F-50D2-4229-B20F-F51CBE23AA4F, OS:11.2, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:F7B4DE64-A88F-4ED9-90BE-AE6C94B04588, OS:11.3, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:F9D848B7-4A9C-4923-8524-87B08078FBE8, OS:12.2, name:iPhone 7 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:BF21FA99-4ACF-432B-B89B-F94B40D3DC5D, OS:11.0.1, name:iPhone 8 }
[ERROR] :  		{ platform:iOS Simulator, id:8E844D06-9661-414B-AE40-38E0CC0699A7, OS:11.1, name:iPhone 8 }
[ERROR] :  		{ platform:iOS Simulator, id:C6997E3A-EA5A-4C0A-A719-43CE51E5096F, OS:11.2, name:iPhone 8 }
[ERROR] :  		{ platform:iOS Simulator, id:D621690F-50EA-446B-B9E7-C4FAD365F0AF, OS:11.3, name:iPhone 8 }
[ERROR] :  		{ platform:iOS Simulator, id:06CEE3D8-68D2-41EA-8FA3-60C2AFC5CE7D, OS:12.2, name:iPhone 8 }
[ERROR] :  		{ platform:iOS Simulator, id:9A5E092D-50D7-4392-9F86-06312348D449, OS:13.0, name:iPhone 8 }
[ERROR] :  		{ platform:iOS Simulator, id:4B6B2815-0E9D-4600-990C-9E0F366B9C07, OS:11.0.1, name:iPhone 8 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:D7767F6F-E1E9-4E33-82AE-2C939029C8C1, OS:11.1, name:iPhone 8 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:94886CCE-7943-4DA6-BA85-C57B3177FD68, OS:11.2, name:iPhone 8 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:AC5804C4-BC6F-49D7-BAF9-C97338745629, OS:11.3, name:iPhone 8 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:12328B3B-7808-4F11-886E-90223B21D3C2, OS:12.2, name:iPhone 8 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:670A85FF-B795-4D8D-8F6F-189C804B7D1E, OS:13.0, name:iPhone 8 Plus }
[ERROR] :  		{ platform:iOS Simulator, id:D5CAC722-934F-442D-BEF3-51B3A062ABB8, OS:13.0, name:iPhone 11 }
[ERROR] :  		{ platform:iOS Simulator, id:3FCA1277-23AE-424C-AA1B-4B6E219F84C1, OS:13.0, name:iPhone 11 Pro }
[ERROR] :  		{ platform:iOS Simulator, id:9C2ACC02-75B4-4462-A49E-B8D2D4F46FA5, OS:13.0, name:iPhone 11 Pro Max }
[ERROR] :  		{ platform:iOS Simulator, id:DC0D71E5-17F2-440B-92A0-0A3EC82C2D62, OS:10.0, name:iPhone SE }
[ERROR] :  		{ platform:iOS Simulator, id:ED567C9A-AE17-4307-9673-F49412AF573E, OS:10.3.1, name:iPhone SE }
[ERROR] :  		{ platform:iOS Simulator, id:6F8EEE93-A05C-423D-B1AE-5E26A0B9E221, OS:11.0.1, name:iPhone SE }
[ERROR] :  		{ platform:iOS Simulator, id:4F2D089F-0BAF-4BB7-B8AE-9A53B42FD12E, OS:11.2, name:iPhone SE }
[ERROR] :  		{ platform:iOS Simulator, id:DFA068CD-A5B7-41CB-9B45-82EAF8798B0A, OS:11.3, name:iPhone SE }
[ERROR] :  		{ platform:iOS Simulator, id:6E6074A4-7F39-4210-886D-06D518BCF901, OS:12.2, name:iPhone SE }
[ERROR] :  		{ platform:iOS Simulator, id:D5A907FD-6C26-4728-A449-0A92C17FC528, OS:11.0.1, name:iPhone X }
[ERROR] :  		{ platform:iOS Simulator, id:A373DD3C-7C50-4979-B04F-AFE1444D1F16, OS:11.2, name:iPhone X }
[ERROR] :  		{ platform:iOS Simulator, id:AF7A65D1-DC09-490F-92A6-78CB1F11E7A1, OS:11.3, name:iPhone X }
[ERROR] :  		{ platform:iOS Simulator, id:9A813B0F-F66E-4E01-853E-46BBF4BC231D, OS:12.2, name:iPhone X }
[ERROR] :  		{ platform:iOS Simulator, id:0D8C55A2-9393-4994-9F13-3F323CF2ACEE, OS:12.2, name:iPhone Xs }
[ERROR] :  		{ platform:iOS Simulator, id:E55C87C7-B75E-4079-AF73-E6D87D991B0D, OS:13.0, name:iPhone Xs }
[ERROR] :  		{ platform:iOS Simulator, id:927576F9-0EE7-4D0C-B34B-2AA852058665, OS:12.2, name:iPhone Xs Max }
[ERROR] :  		{ platform:iOS Simulator, id:2649FC52-245D-4733-9C17-CC5F5AE5CADA, OS:13.0, name:iPhone Xs Max }
[ERROR] :  		{ platform:iOS Simulator, id:5BAFB307-91E7-49B8-A665-708725A29ED2, OS:12.2, name:iPhone Xʀ }
[ERROR] :  		{ platform:iOS Simulator, id:AE7B3692-7480-44AC-8782-97EC6173DA1E, OS:13.0, name:iPhone Xʀ }
[ERROR] :  	Ineligible destinations for the "testClassic" scheme:
[ERROR] :  		{ platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Generic iOS Device }
[ERROR] :  		{ platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Generic iOS Simulator Device }
```

**Comment by cb1kenobi** [comment]
@ssekhri I don't believe it. Did you download the CI build or build from source? If from source, did you do an `npm install`? Please ensure you have selected the correct SDK in the tiapp.xml. Can you verify the ioslib version in the SDK is 1.7.14.

**Comment by ssekhri** [comment]
@cb1kenobi I got the build from the CI. Got the builds again from Jenkins for master and 8_2_X PRs and ensured that the ioslib version is 1.7.14. Same results with SDk builds from both the branch.
I tried on another machine which was just setup new and only XCode 11 has been installed. Didn't see the issue on that machine.
So it might be a scenario that exists on my current macbook which has had different versions of XCode and simulators installed and deleted. Let me know if any further information from my machine could help identify the problem.

**Comment by ssekhri** [comment]
@cb1kenobi tested the PR further. Noticed couple fo scenarios.
The test environment on the macbook had two XCode's under the Applications. The XCode 11 as XCode.app and XCode 10.3 as XCode10. The system has iOS simulators for version 13.0, 12.4 and 12.2.
**Issue 1:**
Step 1: With selected XCode as 11.0 ran a classic project on iOS simulator 13.0 successfully.
Step 2: Next switch the selected XCode to Xcode10.3. Build the project to iOS sim 12.4.  Fails with the logs as below:
```
[TRACE] ** BUILD SUCCEEDED **
[INFO]  Finished building the application in 15s 556ms
[INFO]  Launching iOS Simulator
[TRACE] [ioslib] Selected iOS Simulator: iPhone 8
[TRACE] [ioslib]   UDID    = 095A759C-F3AB-4FA9-9B63-F0AF7E10D33C
[TRACE] [ioslib]   iOS     = 12.4
[TRACE] [ioslib] Autoselected Xcode: 10.3
[TRACE] [ioslib] Checking if the simulator /Applications/Xcode10.app/Contents/Developer/Applications/Simulator.app/Contents/MacOS/Simulator is already running
[TRACE] [ioslib] Simulator is not running
[TRACE] [ioslib] Tailing iPhone 8 Simulator system log: /Users/satyamsekhri/Library/Logs/CoreSimulator/095A759C-F3AB-4FA9-9B63-F0AF7E10D33C/system.log
[TRACE] [ioslib] Running: /Applications/Xcode10.app/Contents/Developer/Applications/Simulator.app/Contents/MacOS/Simulator -CurrentDeviceUDID 095A759C-F3AB-4FA9-9B63-F0AF7E10D33C
[TRACE] [ioslib] iPhone 8 Simulator has exited with code 0
[TRACE] [ioslib] Exited with code: 0
```

**Issue 2:**
Step 1: With selected XCode as 10.3 ran a cleaned up classic project on iOS simulator 12.4 successfully.
Step 2:  Now try to run the same project on iOS simulator 12.2. It hangs during the simulator launch stage with logs as below:
```
[TRACE] ** BUILD SUCCEEDED **
[INFO]  Finished building the application in 14s 284ms
[INFO]  Launching iOS Simulator
[TRACE] [ioslib] Selected iOS Simulator: iPhone Xʀ
[TRACE] [ioslib]   UDID    = 9C5D0B01-26FC-4916-9E6C-505F2CE96BC7
[TRACE] [ioslib]   iOS     = 12.2
[TRACE] [ioslib] Autoselected Xcode: 10.3
[TRACE] [ioslib] Checking if the simulator /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app/Contents/MacOS/Simulator is already running
[TRACE] [ioslib] Simulator is not running
[TRACE] [ioslib] Tailing iPhone Xʀ Simulator system log: /Users/satyamsekhri/Library/Logs/CoreSimulator/9C5D0B01-26FC-4916-9E6C-505F2CE96BC7/system.log
[TRACE] [ioslib] Running: /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app/Contents/MacOS/Simulator -CurrentDeviceUDID 9C5D0B01-26FC-4916-9E6C-505F2CE96BC7
```
Step 3: Again try to build the project to iOS simulator 12.2. On the second attempt the project build successfully to the 12.2 sim.
Step 4: Now try to build the project to the iOS simulator 12.4 (same as in step 1). It fails. The logs are similar as in Issue 1.

**Comment by ssekhri** [comment]
FR Passed. Concurrent builds on iOS simulator work fine. Apart from couple of scenarios that seem to be specific to one system the iOS simulator builds work fine.
Verified On:
Mac OS: 10.14.5
SDK: 8.3.0.v20190927124747
Appc CLI: 7.1.1
JDK: 1.8.0_162
Node: 10.5.0
Studio: 5.1.4.201909061933
Xcode: 11.0, 10.3
Device: iOS simulator 13.0, 12.4, 12.2

**Pull request merged**